creator/ingest_runs/migrations/0001_initial.py

```diff
@@ -11,7 +11,6 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
-        ("files", "0022_add_genomic_workflow_output_type"),
```

Comment:
I don't think this migration should have changed. Is this an unintentional change?

Reply:
This line appeared in the migration initially as an unintended side effect of providing a temporary default value for `id` when prompted by django. It had to be removed for the migrations to work